batch/sql/add-inst-colls.sql

```diff
@@ -0,0 +1,433 @@
+CREATE TABLE IF NOT EXISTS `inst_colls` (
+  `name` VARCHAR(255) NOT NULL,
+  `pool` BOOLEAN NOT NULL,
+  `boot_disk_size_gb` BIGINT NOT NULL,
+  `max_instances` BIGINT NOT NULL,
```

Comment:
I changed the schema of this because all inst_colls need a boot disk and a size.